### ISSUE TITLE
ngx-read-more throwing error?

### ISSUE THREAD

**Issue by alokkarma** [Oneia/read-more-angular2#7]
Node :  v 8.11.2
NPM:  v 5.6.0
Angular cli  :  v 6.0.8
angular : v 6
rxjs : v 6.2.1
typescript : v2.7.2

Is it dependent on rxjs 5 ?

node_modules/ngx-read-more/node_modules/@angular/core/src/application_ref.d.ts(8,10): error TS2305: Module '"C:/Users/alokv/Perforce/alokv_rsh_report/DevOps/analyticsApps/rsh_reports-app/web-src/node_modules/rxjs/Observable"' has no exported member 'Observable'.
node_modules/ngx-read-more/node_modules/@angular/core/src/event_emitter.d.ts(8,10): error TS2305: Module '"C:/Users/alokv/Perforce/alokv_rsh_report/DevOps/analyticsApps/rsh_reports-app/web-src/node_modules/rxjs/Subject"' has no exported member 'Subject'.
node_modules/rxjs/Observable.d.ts(1,15): error TS2307: Cannot find module 'rxjs-compat/Observable'.
node_modules/rxjs/Subject.d.ts(1,15): error TS2307: Cannot find module 'rxjs-compat/Subject'.

**Comment by Oneia** [comment]
yeah. I hope I'll find time to update the directive soon but for now It'll better to copy-paste code and fix it by yourself.

**Comment by Oneia** [comment]
nope. The directive doesn't have rxjs in dependencies. Anyway, I hope I'll find time to update the directive soon.
